torch/_utils.py

```diff
@@ -177,7 +177,7 @@ def _get_async_or_non_blocking(function_name, non_blocking, kwargs):
 # be a TypedStorage
 def _rebuild_tensor(storage, storage_offset, size, stride):
     # first construct a tensor with the correct dtype/device
-    t = torch.tensor([], dtype=storage.dtype, device=storage._untyped_storage.device)
+    t = torch.empty((), dtype=storage.dtype, device=storage._untyped_storage.device)
```

Comment:
```suggestion
    t = torch.empty((0,), dtype=storage.dtype, device=storage._untyped_storage.device)
```
Here and below, otherwise you're creating a scalar Tensor.